index.js

```diff
@@ -89,13 +89,17 @@ lib.one = function (iface, callback) {
         }
         if (!ifaces[iface]) {
             if (typeof callback === 'function') {
-                callback("no interfaces found", null);
+                process.nextTick(function() {
+                    callback(new Error("no interfaces found"), null);
```

Comment:
Oh, also - you should use errors so people have the option of inspecting the stack trace.